### DISCUSSION TITLE
Menu item disabled accessible

### DISCUSSION THREAD

**Discussion by andredewaard** [tailwindlabs/headlessui#2693] [Ideas]
i'm trying to implement some functionality with the Menu component but sometimes a function is 'disabled'. Right now when a button is disabled HeadlessUI will not make it focusable. But accessible wise it would be better if it is focusable but it will describe why you cant interact with it. Im thinking about showing a tooltip on hover/focus or a dialog on click/enter to why this button is disabled. 
Or is there a better way of doing this? 
if i just disable the button some user may never find this functionality because its would not be focusable when they dont do certain actions before.

**Comment by jkjustjoshing** [comment]
From [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/menu_role):

> Menu items include menuitem, menuitemcheckbox, and menuitemradio. **Disabled menu items are focusable but cannot be activated**. [emphasis mine]
